### ISSUE TITLE
Application hang on exit with grpc-js v1.2.0+

### ISSUE THREAD

**Issue by bestbeforetoday** [grpc/grpc-node#1825]
### Problem description
A Node.js client application that makes use of gRPC services exits cleanly with @grpc/grpc-js v1.1.8. Moving to @grpc/grpc-js v1.2.0 or later with no other changes to client application code or libraries results in the application hanging on exit.

### Reproduction steps
The application is relatively complex, invoking gRPC services that include simple request/response, streaming responses, and bi-directional streams. I don't know if there are any specific service invocation types that are triggering this issue, but the invocation and clean-up code in the client application is identical. The only change is the @grpc/grpc-js version used.

### Environment
 - MacOS 11.4, but also observed on Ubuntu build systems
 - Node v12.22.1 and v14.17.1
 - Node installed with nvm locally
 - @grpc/grpc-js v1.2.0 and later (including v1.3.3)

### Additional context
Using wftnode to investigate the state of open handles at completion of the application code gives the following output for @grpc/grpc-js v1.2.0, and the process does not exit afterwards:
```
- File descriptors: (note: stdio always exists)
  - fd 1 (tty) (stdio)
- Sockets:
  - ::1:52359 -> ::1:8080
- Intervals:
  - (120000 ~ 2 min) (anonymous) @ /.../node_modules/@grpc/grpc-js/src/subchannel.ts:291
  - (120000 ~ 2 min) (anonymous) @ /.../node_modules/@grpc/grpc-js/src/subchannel.ts:291
```

With @grpc/grpc-js v1.1.8 it gives the following output, and the process exits cleanly immediately afterwards:
```
- File descriptors: (note: stdio always exists)
  - fd 1 (tty) (stdio)
- Sockets:
  - ::1:52468 -> ::1:8080
- Intervals:
  - (120000 ~ 2 min) (anonymous) @ /.../node_modules/@grpc/grpc-js/src/subchannel.ts:277
```

**Comment by murgatroid99** [comment]
Would you happen to be using the `grpc.keepalive_permit_without_calls` client option? Support for it was added in grpc-js 1.2.0 and I think it might be keeping things alive overzealously.

**Comment by bestbeforetoday** [comment]
Thank you for the quick reply. And yes, digging into the configuration it looks like that setting has been sneaked in as a default. Disabling `grpc.keepalive_permit_without_calls` allows my client application to exit cleanly using @grpc/grpc-js v1.3.3

**Comment by murgatroid99** [comment]
I have published a change that I think will fix this in version 1.3.4

**Comment by bestbeforetoday** [comment]
v1.3.4 appears to fix the application exit issue with `grpc.keepalive_permit_without_calls` for me. 👍
